src/app/index.tsx

```diff
@@ -70,10 +73,24 @@ export default (options: Options) => {
     reducer,
   });
 
+  const language = (typeof window === 'undefined') ? 'en-us' : window.navigator.language;
```

Comment:
i think it would be beneficial to pass the language as an option, browser can get it out of window.navigator and pre-render will either hardcode to english or potentially render different content in different languages